tests/test_chromatic.py

```diff
@@ -1313,6 +1313,9 @@ def evaluateAtWavelength(self, wave):
             ret = ret.shear(g1 = this_shear)
             return ret
 
+        def __repr__(self):
+            return 'galsim.ChromaticGaussian(%r)'%self.sigma
+
```

Comment:
Are there any substantive tests we should add for this?  Something that intentionally draws the same thing multiple times (with different PSFs maybe) to make sure we hit the cache, then don't, then do again. Or something like that?